### DISCUSSION TITLE
When should I use soup vs downloading the latest iso?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#5495] [Unsupported Versions] [answered]
Hi, hope all is well. 
  I saw https://docs.securityonion.net/en/2.3/soup.html. 
  Can I run soup and then rerun setup to possibly fix issues that can cause a previous install to fail?
  I am a newbie and I am wondering when to use soup vs when to just download a new iso?
 It seems like I can use soup to apply the latest updates to the current iso I have and bring this iso up to the latest version and iso? 

thanks for your advice and suggestions

**Comment by n0th1nt0lOOz** [comment, marked as answer]
Run soup is enough to install new update. But only when you complete the last install. Soup can't help you to fix thing not installed yet.

**Reply by iqworks** [reply]
I think you are saying that once you install an iso, you can use soup to install updates on that iso. But if a new iso becomes available with new features, then you have to install this new iso to get these new features. Soup cannot update the old iso with the new features that the new iso has. So soup can only apply updates to features that an iso has. Soup does not add new features to an iso, it only updates the features that are on the iso you run soup on?

**Reply by n0th1nt0lOOz** [reply]
because we use docker and every feature provided via docker so soup could help to update all new feature

**Comment by TOoSmOotH** [comment]
The only reason you would use the ISO to update is if you chose an airgap install. There is no difference between the code on the ISO and the code on github. The only difference in airgap is it is pulling the dockers and the salt code from the iso vs the internet.
